src/Connector.js

```diff
@@ -27,74 +17,76 @@ class Connector {
   }
 
   async connectClient() {
-    this.client = await this.createConnection();
+    this.client.connect();
```

Comment:
Since this function isn't using await anymore, is `async` needed?

Reply:
Good catch! I missed the `await` statement :+1: